src/renderer/dataManager.ts

```diff
@@ -84,9 +90,7 @@ export class DataManager<T extends YatsugiGroup | YatsugiTool> {
 
     async returnItem(toolIDs: string[]) {
         const isValid = await this.validReturnItem(toolIDs);
-        if (!isValid) {
-            throw "もう既に返却されている器材を返却しようとしています。";
-        }
+        if (!isValid) return;
```

Comment:
If an `async` function is finished unexpectedly, we must throw something.
Please revert this change and delete `await showErrorDialog(err);` from "returnTool.tsx".

```typescript
async returnButtonClicked() {
        try {
            await toolData.returnItem(this.state.tools.map(tool => tool.id));
            await showInfoDialog("返却完了しました。");
            this.props.history.push("/");
        }
        catch (err) {
            await showErrorDialog(err);
        }
    }
```